mon-pix/tests/unit/services/storage-test.js

```diff
@@ -0,0 +1,40 @@
+import { setupTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+
+module('Unit | Service | storage', function (hooks) {
+  setupTest(hooks);
+  test('it exists', function (assert) {
+    // when
+    const service = this.owner.lookup('service:storage');
+    // then
+    assert.ok(service);
+  });
+  test('setLogin', function (assert) {
+    // given
+    const service = this.owner.lookup('service:storage');
+    const login = 'login';
+    // when
+    service.setLogin(login);
+    // then
+    assert.strictEqual(sessionStorage.getItem('PIX_LOGIN'), login);
+  });
+  test('getLogin', function (assert) {
```

Comment:
Sauter des lignes entre chaque test.

```suggestion
  });

  test('getLogin', function (assert) {
```